src/aliases.h

```diff
@@ -0,0 +1,25 @@
+/******************************************************************************
+ *
+ * Copyright (C) 1997-2023 by Dimitri van Heesch.
+ *
+ * Permission to use, copy, modify, and distribute this software and its
+ * documentation under the terms of the GNU General Public License is hereby
+ * granted. No representations are made about the suitability of this software
+ * for any purpose. It is provided "as is" without express or implied warranty.
+ * See the GNU General Public License for more details.
+ *
+ * Documents produced by Doxygen are derivative works derived from the
+ * input used in their production; they are not affected by this license.
+ *
+ */
+
+#ifndef ALIASES_H
+#define ALIASES_H
+
+#include "qcstring.h"
+
+QCString resolveAliasCmd(const QCString &aliasCmd);
+std::string expandAlias(const std::string &aliasName,const std::string &aliasValue);
```

Comment:
Looks like `expandAlias` is only needed inside `aliases.cpp` so it can be removed here and made static in the implementation.

Reply:
Thanks, All fixed.